integration-tests/actions/seth/actions.go

```diff
@@ -121,20 +187,27 @@ func SendFunds(logger zerolog.Logger, client *seth.Client, payload FundsToSendPa
 		return nil, err
 	}
 
-	gasLimit := uint64(client.Cfg.Network.TransferGasFee)
+	var gasLimit int64
+	gasLimitRaw, err := client.EstimateGasLimitForFundTransfer(fromAddress, payload.ToAddress, payload.Amount)
+	if err != nil {
+		gasLimit = client.Cfg.Network.TransferGasFee
```

Comment:
yeah, there's a mess with types... but even if trivial it's impact would be too big to do it in this ticket as it would impact all usages of Seth, I'd rather do it in a dedicated PR.